tmd/Topology/analysis.py

```diff
@@ -20,6 +20,67 @@ def sort_ph(ph):
     return np.array(ph)[np.argsort([p[0] - p[1] for p in ph])].tolist()
 
 
+def closest_ph(ph_list, target_extent, method='from_above', return_index=False):
```

Comment:
Rename ph to something humans can understand

Reply:
Even `barcodes` would be better

Reply:
It has to be consisted with the rest of the functions. Barcodes refer to a specific representation of pesistent homology. Dunno, @lidakanari, what do you suggest?

Reply:
I won't have time to look at this before the NCx. sorry... will get back to this once the crisis is over.

Reply:
I care more about readability than consistency. With variable names like this, you are really making sure no one outside of you and Lida will use the code. There is absolutely zero chance than people are going to find out that ph = 'persistent homology'. The words does not even appear in the readme !

Reply:
To be honest ph (persistence homology) is the correct term, but barcode (which is only the visual output) is probably the more used term. I don't mind any of them so I'll let you choose :)

Reply:
Or we could just add the term ph in the readme? Tbh I prefer it over barcode.

Reply:
Sure, I can do add the term with the relevant explanation.

Reply:
PH is not an option. It is un-understandable. However I have no objection for persistent_homology.

Reply:
`topological_persistent_homology_method` is better though